dracut/30ignition/module-setup.sh

```diff
@@ -72,6 +74,7 @@ install() {
     install_ignition_unit ignition-setup-base.service
     install_ignition_unit ignition-setup-user.service
     install_ignition_unit ignition-fetch.service
+    install_ignition_unit ignition-fetch-offline.service
```

Comment:
I know this comment is late, but i'm wondering if we shot down the idea of renaming `ignition-fetch` to `ignition-fetch-online` at some point. I don't expect us to change it now, unless others just love the idea.

Reply:
I think to make it mentally easier, my preference is to keep the semantics of `ignition-fetch` the same and introduce a new stage.